Source/AudioProcessing/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+target_sources("${BaseTargetName}"
+    PRIVATE
+        AudioUtils.cpp
+        Granulate.cpp
+        )
```

Comment:
I personally prefer to add the .h files too. There's a lot of code in the headers and it's nicer if they show up in the IDE if you export to Xcode or VS.

Reply:
Added the header files, let me know if I did something wrong since I had no idea you could